modules/Bio/EnsEMBL/Variation/VCFCollection.pm

```diff
@@ -79,6 +79,7 @@ use Bio::EnsEMBL::Variation::Utils::VEP qw(parse_line);
 use Bio::EnsEMBL::Variation::Utils::Sequence qw(get_matched_variant_alleles);
 use Bio::EnsEMBL::Variation::Utils::VariationEffect qw(MAX_DISTANCE_FROM_TRANSCRIPT);
 use Bio::EnsEMBL::Variation::Utils::Constants qw(%OVERLAP_CONSEQUENCES);
+use Bio::EnsEMBL::Variation::Utils::Config;
```

Comment:
Here you can export the variable `%ATTRIBS`, this variable stores the list of clinical significance.
`use Bio::EnsEMBL::Variation::Utils::Config qw(%ATTRIBS);`

Reply:
Just commited this reccomendation.